js/api/Forms.js

```diff
@@ -12,8 +14,17 @@ export function loadForms(survey, callback) {
   if (surveyFormRelations) {
     surveyFormRelations.query().find({
       success: function(results) {
-        storeForms(results)
-        if (callback) callback(null, results, survey)
+        realm = new Realm({schema: [Submission]});
+        newForms = []
+        _.forEach(results, function(form, key){
+          let submission = realm.objects('Submission').filtered(`formId = "${form.id}"`)
+          // Only include the current form if there have been no submissions to it yet.
+          if(submission.length == 0){
+            newForms.push(form);
+          }
+        });
+        storeForms(newForms)
```

Comment:
with realm.io we don't need to 'cache' forms in Store.js as well we should delete this line